### ISSUE TITLE
Add case images and design files, if there are any

### ISSUE THREAD

**Issue by DeflateAwning** [bschwind/key-ripper#55]
I'm assuming you also had a 3d-printed case or similar? You should post info about it! If not, my mistake!

**Comment by bschwind** [comment]
Yes, I did make a case for it! In fact it's still a work in progress which I recently picked up again.

This is quite obscure, but the code for it is currently in this repository:

https://github.com/bschwind/opencascade-rs/blob/keyboard-case-v2/examples/src/keyboard_case.rs

I actually started that project just to be able to define the case in Rust code, but it has turned into its own project which I also want to flesh out.

Here are some previews of it:

<img width="1138" alt="Screen Shot 2023-11-18 at 10 53 37 PM" src="https://github.com/bschwind/key-ripper/assets/458432/b45d07dd-021d-442e-a3e2-746d0b5d5f02">

<img width="1274" alt="Screen Shot 2023-11-18 at 10 53 25 PM" src="https://github.com/bschwind/key-ripper/assets/458432/fa18f4ee-e293-4555-88d7-d7a1761ce2ae">

Once the code in opencascade-rs gets further along and more mature, I'll move the code for the keyboard case into this repo. Currently I often need new features added to opencascade-rs, so I add those alongside development of the keyboard case.

**Comment by DeflateAwning** [comment]
Very neat project, very awesome!

I'm sure you know about OpenSCAD, but check out BOSL2 for OpenSCAD. It has a really nice API (e.g., anchor argument for zcyl, etc.), and could potentially be used as a model as least somewhat for the opencascade-rs project!

**Comment by bschwind** [comment]
Could you explain anchor arguments a bit more and what you like about the API? I think I kind of understand them.

For positioning right now my library mostly uses workplanes, which you can construct directly on a typical plane like XY, or they can be constructed from the face of an object.

I still need to figure out a good API for attaching objects together with constraints though.
